packages/core/src/install/index.ts

```diff
@@ -269,7 +269,7 @@ export async function mutateModules (
       !opts.fixLockfile &&
       installsOnly &&
       (
-        frozenLockfile ||
+        frozenLockfile && !opts.lockfileOnly ||
```

Comment:
is that expected to be written without brackets `()` around it? I didn't test anything, I'm just asking

Reply:
It will work without brackets just fine. && has bigger priority than ||

Reply:
ahh ok I forgot that, cool 👍🏻